src/acl.c

```diff
@@ -2411,6 +2411,20 @@ void addACLLogEntry(client *c, int reason, int context, int argpos, sds username
     }
 }
 
+const char* getAclErrorMessage(int acl_res) {
```

Comment:
i guess we can't easily use that in the "switch-case" in acl.c (it's more verbose).
maybe we should at least add some cross reference comments between them, so that when one is updated the other one will too..

Reply:
Which "switch-case" you refer to?

Reply:
```c
        if (result != ACL_OK) {
            sds err = sdsempty();
            if (result == ACL_DENIED_CMD) {
                err = sdscatfmt(err, "This user has no permissions to run "
                    "the '%s' command", cmd->fullname);
            } else if (result == ACL_DENIED_KEY) {
                err = sdscatfmt(err, "This user has no permissions to access "
                    "the '%s' key", c->argv[idx + 3]->ptr);
            } else if (result == ACL_DENIED_CHANNEL) {
                err = sdscatfmt(err, "This user has no permissions to access "
                    "the '%s' channel", c->argv[idx + 3]->ptr);
            } else {
                serverPanic("Invalid permission result");
            }
```

Reply:
+1 about server panic when a value is provided outside the known enum types.

Reply:
Added comment on both locations.